conda/recipes/pynvjitlink/meta.yaml

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2018-2024, NVIDIA CORPORATION.
+
+{% set data = load_file_data("pyproject.toml") %}
+
+package:
+  name: pynvjitlink
+  version: {{ data.get("project", {}).get("version", "") }}
+
+source:
+  path: ../../..
+
+build:
+  script:
+    - {{ PYTHON }} -m pip install . -vv
+  ignore_run_exports_from:
+    - libnvjitlink-dev
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - cuda-version {{ cuda_compiler_version }}
```

Comment:
Right now this is basically set so that we'll always use the latest major version available on cf. Since this is our short-term solution until we figure out how to use non-conda packages to get the latest CTK, should we be pinning more precisely for now to exactly the build we want, or do we want to let this float across minor versions?

Reply:
I'm happy to let it float across minor versions until we can separate this from conda-forge entirely, per the ["long term plan."](https://github.com/rapidsai/pynvjitlink/pull/33#discussion_r1449272016)

Reply:
Actually as `{{ compiler('cuda') }}` already comes with `cuda-version`, we could drop this from `build`

```suggestion
```

Reply:
How would we be setting the cuda compiler version without this line?

Reply:
Via `cuda_compiler_version` in `conda_build_config.yaml`. This is how it [works in conda-forge]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/cbccb41e313c017c1ad15db6c287e87aa57d54dd/recipe/conda_build_config.yaml#L44-L47 ) as well

Reply:
Gotcha. Then yes, I'll remove this line.